rasa/core/brokers/pika.py

```diff
@@ -304,6 +319,22 @@ def publish(
             "'{}':\n{}".format(self.queue, self.host, body)
         )
 
+    @property
+    def _message_properties(self) -> "BasicProperties":
+        """Create RabbitMQ message properties.
+
+        Returns:
+            pika.spec.BasicProperties with the `RASA_ENVIRONMENT` environment
+            variable as the properties' app_id value. If this variable is unset, empty
+            pika.spec.BasicProperties.
+
+        """
+        from pika.spec import BasicProperties
+
+        kwargs = {"app_id": self.rasa_environment} if self.rasa_environment else {}
```

Comment:
why don't we always set the `app_id` and have e.g. `rasa` as fallback in case `RASA_ENVIRONMENT` is not set?

Reply:
well the logic on the other end assigns `RASA_PRODUCTION` if the `app_id` is _not set_

Reply:
Ok 👍 Setting this to `production` in the open-source part probably doesn't make that so much sense.